Support for drop rejects with multiple rule-action sets

bperel opened the pull request.

Commit:
Fixed to work in IE7 and IE8.

Commit:
Replaced $.browser references with a regex lookup on navigator.userAgent, since $.browser is deprecated in jQuery 1.9.

Commit:
Fixed a problem where the browser version regex was seeing IE 10 as IE 1.

Commit:
Attached events to $(document) always, instead of $(windows) for some browsers. Removed the browser detection and switched the event.button to event.which. Thanks to estmatic for the advice!

bperel committed:
Support for accept function and revert action

bperel committed:
Fix element passed to the revertAction function

bperel committed:
Support for multiple rule-action sets for drop rejects

bperel committed:
Merge branch 'master' of git://github.com/jasonruesch/Nestable

bperel committed:
Fix use of apply on IE 8

bperel committed:
Fix multiple rule handling

sebVier commented:
Hi @bperel , i know this is an old branch but fit's my current project quite well. Do you know if it's also possible to reject something based on the destination it's moved to?
For example i want to prevent people dropping items in the root of my tree but all other 'levels' are OK. 
1-2
--2-3
--2-3
--2-3-4
In the above example lvl1 would be root, so offlimit's. But all others are fine so something from lvl4 could be dropped in lvl2 etc. etc. 
having a hard time with this, hope you could share some insights :).
Thanks in advance!

bperel commented:
@sebVier I'll check this in the next days

sebVier commented:
Awesome thanks a lot!

bperel committed:
Pass dragged element to rejection filter function

bperel committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	jquery.nestable.js

bperel committed:
Add example of rejection rule based on currently dragged item

bperel committed:
Update Demo links

bperel commented:
@sebVier I had a look at this. You may have been confused by the inaccurate comments that I put on my code. I fixed that and updated from upstream into https://github.com/bperel/Nestable. Demo is on http://bperel.github.com/Nestable/

sebVier commented:
@bperel Thank you so much! work's great now!

jerviscui commented:
@bperel  so good.
I have implemented a method like reject drop,
after that I see your fork.  :smile: :smile: :smile:

bilalyilmax commented:
@bperel Thanks you for this solution. :+1:

bperel committed:
Update reference to jquery.js
